dev-python/logbook/logbook-1.5.3.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="A logging replacement for Python"
+HOMEPAGE="https://logbook.readthedocs.io/en/stable/
+	https://github.com/getlogbook/logbook
+	https://pypi.org/project/Logbook/"
+SRC_URI="https://github.com/getlogbook/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+distutils_enable_tests pytest
+distutils_enable_sphinx docs
+
+BDEPEND="
+	test? (
+		app-arch/brotli[${PYTHON_USEDEP},python]
+		dev-python/execnet[${PYTHON_USEDEP}]
+		dev-python/jinja[${PYTHON_USEDEP}]
+		dev-python/pyzmq[${PYTHON_USEDEP}]
+		dev-python/redis-py[${PYTHON_USEDEP}]
```

Comment:
Is this actually used during the tests, given that you remove the test below?

Reply:
That is a good question. And I will try to find the answer :-)